### PR TITLE
Adds Pilot's Seat to all escape shuttles. Escaping in this chair means additional spacebux

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
This PR made at the request of @UrsulaMejor 

Adds pilot's seat to all escape shuttles. Being buckled into this seat when the round ends grants a pilot's bonus in the form of double spacebux wages.

Note that the bonus applies prior to the reduction of wages for late join. This means you can still join last minute and earn a full round's wages worth of bux in the form of your bonus...if you can get the chair.

image

also fixes an issue of displaying incorrect base wages when nukeops / wizards escaped.

## Why's this needed? 
To make people fight over the best chair

## Changelog

```
(u)Sovexe and UrsulaMajor:
(*)Added pilot's seat to all escape shuttles. Being buckled into this seat when the round ends grants a pilot's bonus in the form of additional spacebux earnings.
```

### CODE REVIEW

**Comment by LanceFyre99** [comment]
Would the AI be able to benefit from this? I've seen the AI get the pilot's seat several times and it would be a shame for them to get left out of any potential bonus

**Comment by Sovexe** [comment]
Currently only mobs capable of buckling Into the chair may receive the bonus

**Comment by Kyle2143** [comment]
I think that doubling is too much, a flat rate would be better, like 300-600. It's funny and worth trying out, but I worry that such a change would cause fights and arguments for meta reasons. I already hear people complain about dying and such in the vein of spacebux rewards.

**Comment by Sovexe** [comment]
Per Kyle's feedback I've have set the pilot's bonus at a flat 500 bux

image

**Comment by ZeWaka** [comment]
Can you add a special case for the AI?

**Comment by Sovexe** [comment]
as request the AI can now be the pilot by being on the same tile as the captain's seat while no one else is buckled in. Additionally the AI can be in its AEye form anywhere, so long as it's actual mainframe is in the pilots seat.

File: code/datums/gameticker.dm [hunk 1]
```diff
@@ -684,6 +685,26 @@ var/global/current_state = GAME_STATE_WORLD_INIT
...
+						bank_earnings.pilot = 1
+						earnings += pilot_bonus
+				else if (isAI(player))
+					var/mob/living/silicon/ai/M = null
+					if (isAIeye(player))
+						M = player:mainframe
+					else
+						M = player
+					for (var/i in M.loc.contents)
+						if (istype(i,/obj/stool/chair/comfy/shuttle/pilot))
```
Comment (pali6) [inline code/datums/gameticker.dm:702]: Coul be replaced by something like `var/obj/stool/chair/comfy/shuttle/pilot/i = locate() in src.loc`.

Comment (Sovexe) [reply code/datums/gameticker.dm]: done

**Pull request merged**